uk.co.compucorp.civicrm.hrleaveandabsences/views/shared/directives/leave-request-popup.html

```diff
@@ -11,14 +11,14 @@ <h4 class="modal-title">
 </div>
 <crm-loading show="!$ctrl.loading.absenceTypes">
   <div
-    ng-show="$ctrl.isRole('manager')"
+    ng-show="$ctrl.isRole('manager') && !$ctrl.isMode('create')"
```

Comment:
why this change? why do you need to show this bit only when the role is manager *and* when we are creating a new leave request?